images/rook/Dockerfile

```diff
@@ -15,6 +15,7 @@
 FROM BASEIMAGE
 
 ADD rook /usr/local/bin/
+COPY flexvolume /
```

Comment:
please use `ADD rook flexvolume /usr/local/bin`. no need for a new layer.

Reply:
sure